flink-ml-core/src/main/java/org/apache/flink/ml/param/ParamValidators.java

```diff
@@ -38,6 +38,22 @@ public static <T> ParamValidator<T> gtEq(double lowerBound) {
         return (value) -> value != null && ((Number) value).doubleValue() >= lowerBound;
     }
 
+    // Checks if all values in the parameter numericalArray is greater than or equal to lowerBound.
```

Comment:
nit: `numericalArray` is not a Proper Noun in this context, as we do not have any class with this name. Consider expressing it in other ways, like "array-typed parameter" or "numerical array parameter".